### PR TITLE
Create SECURITY.md

### PR DESCRIPTION
**Pull request by lcarva**

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Security Policy
+
+## Reporting a Vulnerability
+
+To report a security issue, please click on the "Report a vulnerability" button from the "Security" tab. 
+See detailed [instructions]([url](https://docs.github.com/en/code-security/security-advisories/guidance-on-reporting-and-writing/privately-reporting-a-security-vulnerability#privately-reporting-a-security-vulnerability)https://docs.github.com/en/code-security/security-advisories/guidance-on-reporting-and-writing/privately-reporting-a-security-vulnerability#privately-reporting-a-security-vulnerability).
+
+The maintainers will respond within 3 working days of your email.
```
Comment (zregvart) [inline SECURITY.md:8]: Won't we respond on the issue? For email we'd need to have a PGP key published here...

**Review by zregvart — COMMENTED** [review]
Do we need a sentence about acknowledgements?

File: SECURITY.md (hunk 1 above)
Comment (lcarva) [reply SECURITY.md]: Oops. Bad copy-and-paste. Updated.

**Comment by lcarva** [comment]
> Do we need a sentence about acknowledgements?

Can you elaborate?

**Review by robnester-rh — APPROVED** [review]
LGTM

**Comment by zregvart** [comment]
> > Do we need a sentence about acknowledgements?
> 
> Can you elaborate?

Folk usually want an acknowledgment for the vulnerability discovery. It is common to mention them on the CVE and to have a “acknowledgments” page where they are thanked for reporting responsibly. So a sentence saying we will fully acknowledge the reporter for responsibly disclosing the vulnerability, or the like, would be nice.

**Comment by lcarva** [comment]
> we will fully acknowledge the reporter for responsibly disclosing the vulnerability

Thanks! Added your suggestion.

**Review by zregvart — APPROVED** [review]

**Pull request merged**